content/en/blog/_posts/2024-08-22-cpumanager-static-policy-distributed-cpu-across-cores/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+title: 'Kubernetes v1.31: New Kubernetes CPUManager Static Policy: Distribute CPUs Across Cores'
+date: 2024-08-22
+slug: cpumanager-static-policy-distributed-cpu-across-cores
+author: >
+  [Jiaxin Shan](https://github.com/Jeffwan) (Bytedance)
+---
+
+In Kubernetes 1.31, we are excited to introduce a significant enhancement to CPU management capabilities: the `distribute-cpus-across-cores` option for the CPUManager static policy. This feature is currently in alpha and hidden by default, marking a strategic shift aimed at optimizing CPU utilization and improving system performance across multi-core processors.
+
+## Understanding the Feature
+
+
+Traditionally, Kubernetes' CPUManager tends to allocate CPUs as compactly as possible, typically packing them onto the fewest number of physical cores. However, allocation strategy matters, CPUs on the same physical host still share some resources of the physical core, such as the cache and execution units, etc.
+
+<img src="cpu-cache-architecture.png" alt="cpu-cache-architecture" class="left" style="max-width: 20em; margin: 1em" >
```

Comment:
I see some other post use this way to embed the image. can i know any other `class` options before `left` available? @rashansmith

Reply:
I think its recommended to use the figure shortcode:

```{{< figure src="IMAGENAME.png" caption="description of your image" >}}```

The way the image is now looks too small on the preview to be seen:https://deploy-preview-47013--kubernetes-io-main-staging.netlify.app/blog/2024/08/22/cpumanager-static-policy-distributed-cpu-across-cores/

Reply:
Got you. I updated to use `figure`